### PR TITLE
webgui: more functionality in openui5 canvas

### PR DESCRIPTION
**Pull request by linev**
New components in GED editor
Show inspector in openui5 dialog
Special dialog for menu commands arguments
Dummy FitPanel 
Many bugfixes and improvements in JSROOT itself

### CODE REVIEW

File: etc/http/openui5/controller/Canvas.controller.js [hunk 1]
```diff
@@ -79,94 +161,120 @@ sap.ui.define([
...
+
+         if (name != "Fit panel") return;
+
+         var curr = this.getView().getModel().getProperty("/LeftArea");
+
+         this.showLeftArea(curr == "FitPanel" ? "" : "FitPanel");
+		},
+
+		showMessage : function(msg) {
+		   MessageToast.show(msg);
```
Comment (bellenot) [inline etc/http/openui5/controller/Canvas.controller.js]: Could you adjust the code indentation inside `onToolsMenuAction : function(oEvent)`?

File: etc/http/openui5/view/FitPanel.view.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    	<footer>
+    	   <Bar>
+    			<contentRight>
+    				<Button id="fitpanel_Fit" text="Fit" press="handleFitPress" />
+    				<Button id="fitpanel_Close" text="Close" press="handleClosePress" />
+    			</contentRight>
+    		</Bar>
+    	</footer>
+   </Page>
+</mvc:View>
```
Comment (bellenot) [inline etc/http/openui5/view/FitPanel.view.xml]: Same here: code indentation (try to have a constant number of spaces...)

**Comment by linev** [comment]
Bertrand,

can we postpone formatting for later.
It is pure prototype and will be changed many times

**Comment by bellenot** [comment]
OI, then fine with me, but the xml files should be corrected for the next PR

**Comment by linev** [comment]
> OI, then fine with me, but the xml files should be corrected for the next PR

Bertrand, you are free to submit required changes to jsroot repository.
Here is just copy of dev branch.

**Pull request merged**